src/Conversions/Commands/RegenerateCommand.php

```diff
@@ -75,16 +76,25 @@ public function handle(MediaRepository $mediaRepository, FileManipulator $fileMa
     public function getMediaToBeRegenerated(): Collection
     {
         $modelType = $this->argument('modelType') ?? '';
+
         $mediaIds = $this->getMediaIds();
+        $noIds = count($mediaIds) === 0;
```

Comment:
This whole bit feel unreadable to me. 

Try to start with the special cases (so starting with determining if you should start from an id or model.

Instead of using `&&`, introduce new helper functions where you don't use `&&` but work with early returns.

Also make it possible to get models that start with an id greater than some id.